src/umpire/ResourceManager.hpp

```diff
@@ -328,6 +328,7 @@ class ResourceManager {
   util::AllocationMap m_allocations;
 
   std::list<std::unique_ptr<strategy::AllocationStrategy>> m_allocators;
+  std::vector<std::string> m_shared_allocators_by_name;
```

Comment:
```suggestion
  std::vector<std::string> m_shared_allocator_names;
```

The `by_name` suggests it's a structure to access the allocators by their names (like a map). This isn't the case, so I suggest a slightly different name.